### ISSUE TITLE
Homey offline (crash?) after deleting Z-wave device

### ISSUE THREAD

**Issue by djanssen1** [athombv/homey#1724]
**What is your Homey version (Settings → System)?**

1.5 RC2

**What did you try to achieve?**

Delete a Z-wave device. It showed the exclusion info. Then device was triggered for exclusion and I saw that Homey detected device exclusion and then it went offline and wasn't reachable anymore

**Please attach a Homey System log (Settings -> System -> Send Log)**

EBC975BABB

**Comment by djanssen1** [comment]
After reboot, I worked again on excluding devices and same behaviour was there. But this time lots of offline messages came but still was able to access Homey. I have attached a new log

4565FEC0C0

**Comment by CyberSponk** [comment]
Same with ZigBee-Devices. After some Minutes Offline Homey was reachable again.

**Comment by philsniff** [comment]
here also, removed netatmo device and suddently homey went offline!
will wait if it comes back...

**Comment by philsniff** [comment]
and deleting the app under settings also freezes homey desktop and hemey goes offline

**Comment by DeefK** [comment]
Same here just happend with a netatmo device. Deleted the device, then pressed + button to add a new device and homey goes offline. It did come only after several minutes.

**Comment by daneedk** [comment]
Same here after removing a Aeotec Multisensor 6 homey became unresponsive for almost 5 minutes.
1.5.0 rc5.

**Comment by bvdbos** [comment]
Maybe it's because of deleting the insights which takes a lot of processing-power?

**Comment by jeroenvollenbrock** [comment]
Is this still happening on rc5? I'd appreciate a log on rc5 (taken as soon as possible after the issue occured)

**Comment by djanssen1** [comment]
Yes still happening, just tested it by deleting a device, it showed 3 times offline message and did not refresh devices page. But it remain responsive. In RC2 Homey was not reachable.

Log: 315B2BDF24

**Comment by jeroenvollenbrock** [comment]
Should be fixed in rc6

**Comment by bvdbos** [comment]
@djanssen1 Does this work for you now? @CyberSponk says it works ofr Zigbee...

**Comment by djanssen1** [comment]
So I tried to see if this issue was resolved... 
tried to remove a z-wave device that was not live anymore

1st try: vdevice.drivers.zwavebasic.pair.invalid_result
2nd try: No device for removal found....
3th try: succesfull removed but not reflected in UI

after a minute or so... Homey went offline...
After some time I did a PTP

Then it came back online but gave error with logging in "unautorized" so basically Homey got corrupted and need to go to factory default... losing 80 z-wave devices and more. But that is risk of going experimental with a product that does not have a backup feature.

So no logs available...

**Comment by jeroenvollenbrock** [comment]
Closing as duplicate of #1815
